.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [ push, pull_request, pull_request_review, pull_request_review_comment ]
```

Comment:
The comment above says this triggers only on the master branch? Will it also trigger for the dev-branch? Might make sense to trigger it also for all other branches, so stuff doesn't need to be merged into dev before running tests automatically?

Reply:
We forgot to edit the comment when we edited this line. It actually triggers for every single branch.
Else we would need to set a property ```branch: [ master ] ``` for each of the triggers.